test/quoted.test.js

```diff
@@ -0,0 +1,57 @@
+/*
+  Copyright © 2018 Andrew Powell
```

Comment:
Let me know if you'd to add your name to this `Copyright`.  This entire file is your work, so it's totally your call if you'd like to take credit.

Reply:
There is no need for this. I basically copied other test files. 
Thanks anyway😊!